platform/windows/detect.py

```diff
@@ -269,50 +203,37 @@ def configure(env):
         env.Append(CCFLAGS=["/I" + p for p in os.getenv("INCLUDE").split(";")])
         env.Append(LIBPATH=[p for p in os.getenv("LIB").split(";")])
         env.Append(CCFLAGS=["/I" + DIRECTX_PATH + "/Include"])
```

Comment:
Works with MSVC if I remove this line, since `DIRECTX_PATH` definition was removed.